apps/accounts/templates/accounts/activate_account.html

```diff
@@ -1,5 +1,4 @@
 {% autoescape off %}
-    Hi {{ user.full_name }},
-    Please click on the link to confirm your registration,
+    Please click on the link to confirm your registration with email {{ email }}
     http://{{ domain }}{% url 'accounts:activate_account' id %}
```

Comment:
You need to change `http/https` related to the configuration. So, you need also to move it to the settings.